aiosip/protocol.py

```diff
@@ -23,6 +23,9 @@ def send_message(self, msg, addr):
         LOG.log(5, 'Sending to: "%s" via UDP: "%s"', addr, msg)
         self.transport.sendto(msg.encode(), addr)
 
+    def connection_lost(self, error):
+        self.app._connection_lost(self)
```

Comment:
I think we should keep logging the error, at least when it's not `None`. Either here or in `app._connection_lost`. (Same for the other protocol)

Reply:
You already get a log message when the connection drops from Peer._disconnected